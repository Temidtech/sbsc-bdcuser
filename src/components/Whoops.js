import React, { PureComponent } from 'react';
import { View, Text } from 'react-native';

export default class Whoops extends PureComponent {
  render() {
    return (
      <View>
        <Text>{this.props.message}</Text>
      </View>
    );
  }
}
