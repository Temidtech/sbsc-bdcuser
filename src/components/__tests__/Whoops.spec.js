import React from 'react';
import renderer from 'react-test-renderer';
import Whoops from '../Whoops';

describe('<Whoops />', () => {

  it('renders the Whoops component correctly', () => {
    const tree = renderer.create(
      <Whoops message="An error occured" />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
