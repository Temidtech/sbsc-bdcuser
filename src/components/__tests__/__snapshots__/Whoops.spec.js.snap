// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Whoops /> renders the Whoops component correctly 1`] = `
<Text
  accessible={true}
  allowFontScaling={true}
  ellipsizeMode="tail"
  style={
    Object {
      "color": "red",
      "fontFamily": "OpenSans-SemiBold",
    }
  }
>
  An error occured
</Text>
`;
